CHANGELOG.md

```diff
@@ -3,6 +3,31 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## [7.3.1] Preview Release
+### Added
+- Added support for NTLM Authentication [#998](https://github.com/Microsoft/mssql-jdbc/pull/998)
+- Added new connection property 'useFmtOnly' to retrieve parameter metadata [#1044](https://github.com/Microsoft/mssql-jdbc/pull/1044)
+- Added support for JDK 12 and build additional "jre12" JAR [#1050](https://github.com/Microsoft/mssql-jdbc/pull/1050)
+- Added 'keyVaultProviderClientId' and 'keyVaultProviderClientKey' connection properties to enhance Always Encrypted usability [#902](https://github.com/Microsoft/mssql-jdbc/pull/902)
+- Implemented `hashCode()` and `equals()` APIs for `SQLServerDataTable` and `SQLServerDataColumn` [#1025](https://github.com/Microsoft/mssql-jdbc/pull/1025)
+- Added Maven shade plugin configuration to package the driver in uber-jar [#1043](https://github.com/Microsoft/mssql-jdbc/pull/1043) [#1078](https://github.com/Microsoft/mssql-jdbc/pull/1078) [#1081](https://github.com/Microsoft/mssql-jdbc/pull/1081)
```

Comment:
package the driver into uber-jar(s)?

Reply:
I thought we're using the term "shaded"?

Reply:
Shaded is jar suffix, right terminology of these jars is "uber jars" or "fat jars"